### PR TITLE
Items feature

### PR DESCRIPTION
**Pull request by AmbujaAK**
Hi @hungps , 

I had checked PR #48, there is no progress in it from last 3 months. So I decided to complete this pending work of @LucasFab. I took most of his code from the PR and converted Riverpod to bloc myself.

Please take a look at it. 

Thanks to @LucasFab

### CODE REVIEW

**Comment by AmbujaAK** [comment]
Screenshot :

simulator_screenshot_7C7B27B6-2D5E-4009-B8FB-8FFEF8E5E7F6

File: lib/data/source/github/github_datasource.dart [hunk 1]
```diff
@@ -1,14 +1,18 @@
...
 
 import 'package:pokedex/core/network.dart';
+import 'package:pokedex/data/source/github/models/item.dart';
 import 'package:pokedex/data/source/github/models/pokemon.dart';
 
 class GithubDataSource {
+  GithubDataSource(this.networkManager);
+
+  static const String itemsURL =
+      'https://gist.githubusercontent.com/AmbujaAK/1a47a19039a4203252ed834ff65ed140/raw/4058b8ee4a4747ad7bdeb63b69eb40ce15e5abe3/items.json';
```
Comment (hungps) [inline lib/data/source/github/github_datasource.dart:11]: Please use this URL instead
https://gist.githubusercontent.com/hungps/48f4355fb1a89ddaf47f56961dc8a245/raw/pokemon-items.json

**Review by hungps — CHANGES_REQUESTED** [review]
Hi @AmbujaAK, thank you for contributing to this project!

File: lib/data/source/github/github_datasource.dart (hunk 1 above)
Comment (AmbujaAK) [reply lib/data/source/github/github_datasource.dart]: I have changed the url and checked the items. working fine.

**Comment by hungps** [comment]
@AmbujaAK Thank you!

**Review by hugovidafe — COMMENTED** [review]
Displays the category for each item, instead of showing the same name twice

**Comment by hugovidafe** [comment]
> simulator_screenshot_7C7B27B6-2D5E-4009-B8FB-8FFEF8E5E7F6

Also, why are we seeing the name twice instead of the category?

**Pull request merged**